キーボードショートカットによる制御機能の実装 (#242)
MLShukai/pamiq-core: docs/user-guide/console.md

- `resuming` - System is in the process of resuming
- `shutting down` - System is shutting down

## Keyboard Shortcut Controller

> For agents that interact with the desktop or require quick pause/resume capabilities, PAMIQ-Core provides a keyboard shortcut controller.

Reviewer: 何ができるのかに説明をフォーカスしたいかも
```suggestion
PAMIQ-Core provides a keyboard shortcut controller. Users can pause/resume a agent by keyboard shortcuts. 
```

> 

Reviewer: デフォルトのショートカットキーここに表記したいかも　上の説明読んだ後に読み手が気になるのって、どうやってインストールするかというより、具体的にどういう機能があるかな気がする
```suggestion
### Default Shortcuts

- Windows/Linux: `Alt+Shift+P` (Pause) and `Alt+Shift+R` (Resume)
- macOS: `Option+Shift+P` (Pause) and `Option+Shift+R` (Resume)
- Windows also supports `Alt+Shift+Q` to quit the controller
```
